### PR TITLE
homework node-week1

### PR DESCRIPTION
**Pull request by AlejandroGispert**
node express and knex api

### CODE REVIEW

File: api/src/index.js [hunk 1]
```diff
@@ -6,26 +6,82 @@ import knex from "./database_client.js";
...
+  const futureMeals = await knex.raw(
+    `SELECT * FROM Meal WHERE created_date >= NOW() ORDER BY created_date DESC`
+  );
+  if (futureMeals.length > 0) {
+    res.send(futureMeals);
+  } else {
+    res.send([]);
+  }
+});
+app.get("/past-meals", async (req, res) => {
```
Comment (babak-f) [inline api/src/index.js:32]: Try adding vertical spacing between blocks of code, to make the code easier to read.

File: api/src/index.js [hunk 2]
```diff
@@ -6,26 +6,82 @@ import knex from "./database_client.js";
...
 // This nested router example can also be replaced with your own sub-router
 apiRouter.use("/nested", nestedRouter);
 
 app.use("/api", apiRouter);
 
+app.get("/future-meals", async (req, res) => {
+  //   const meals = await knex.raw("SELECT * FROM Meal");
+  // console.log(meals);
+  const futureMeals = await knex.raw(
+    `SELECT * FROM Meal WHERE created_date >= NOW() ORDER BY created_date DESC`
```
Comment (babak-f) [inline api/src/index.js:24]: Here we are using the wrong notation for the string. Can you guess why? ;)

**Review by babak-f — APPROVED** [review]
Good job ;)

File: api/src/index.js (hunk 2 above)
Comment (AlejandroGispert) [reply api/src/index.js:24]: ahh i see, cause it can think is the name of a database `` i better use normal ""

**Pull request merged**